build-system/tasks/helpers.js

```diff
@@ -430,17 +414,17 @@ function compileJs(srcDir, srcFilename, destDir, options) {
         .then(() => {
           if (process.env.NODE_ENV === 'development') {
             if (destFilename === 'amp.js') {
-              return enableLocalTesting(unminifiedRuntimeTarget);
+              return enableLocalTesting('dist/amp.js');
             } else if (destFilename === 'amp-esm.js') {
-              return enableLocalTesting(unminifiedRuntimeEsmTarget);
+              return enableLocalTesting('dist/amp-esm.js');
             } else if (destFilename === 'amp4ads-v0.js') {
-              return enableLocalTesting(unminifiedAdsTarget);
+              return enableLocalTesting('dist/amp4ads-v0.js');
```

Comment:
@zhouyx @calebcordry FYI if you've had problems with local testing of unminified `amp4ads-v0.js`, this is likely going to fix it. Let me know if there are any other files I'm missing here, or in [dist.js](https://github.com/ampproject/amphtml/pull/22135/files#diff-7c22ae941e7d608ea3ed2ca2f101e835).